src/components/cards/List.tsx

```diff
@@ -56,13 +56,13 @@ const List = ({ list: { id, name, status, created_at } }: ListProps) => {
     }
     return (
         <Link to={`/history/${id}`}>
-            <div className="flex w-full p-4 shadow rounded-lg justify-between items-center mb-8 bg-white hover:bg-gray-extra-light transition-colors duration-300">
+            <div className="flex flex-col md:flex-row w-full p-4 shadow rounded-lg justify-between items-center mb-8 bg-white hover:bg-gray-extra-light transition-colors duration-300">
                 {/* List name */}
-                <div className="font-bold">{name}</div>
+                <div className="font-bold mb-2 md:mb-0">{name}</div>
 
-                <div className="flex items-center w-1/2 justify-around">
+                <div className="flex flex-col md:flex-row items-center md:w-1/2 justify-around">
                     {/* Icon + Date */}
-                    <div className="flex items-center">
+                    <div className="flex items-center mb-3 md:mb-0">
                         <MdDateRange
                             style={iconStyle}
                             className="text-gray-light mr-2"
```

Comment:
On screens smaller than 768px, aligning content here on the center looks kind of weird to me, especially since we have content to the left (navbar) and most the text is left-aligned. Also on tablet screens should we display each list as 50% width? That way we can fit more content on tablets, but on mobile, we just use the full width.

Reply:
I'm not sure to understand. On smaller screens, I can't keep the original List design as I have not enough space. And I tried to align everything to the left ( with the flex column ) and It feels kinda weird I think. What do you mean by displaying each list as 50% width? Do you mean to have 2 columns of lists?

Reply:
Yeah, what I meant for the list display on the tablet was to put it into two columns. I guess we can wait with that and see how it looks with a lot of lists (if we leave too much-unused whitespace on the right). Regarding the center-aligned cards, I still think it breaks the flow too much if the content is centered
<img width="468" alt="Screenshot at Sep 16 11-55-07" src="https://user-images.githubusercontent.com/36310604/93323273-b05ada80-f814-11ea-9e99-02b722247513.png">
For stuff like this i usually just like to follow something like material design guidelines, in our example this would me material design cards.

Reply:
I will check the material design cards ;)

Ok to wait for the lists in two columns because I'm not sure of the user experience. Anyway, It was just a first pass, it's not a lot of work to modify things if we change our mind, or just to do some design tests if we want ;)